<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PAÜ Mühendislik Fakültesi Optik Okuyucu Programı</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Soru Sayısı</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="soruSayisi">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="value">
               <number>40</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="otomatikAlgila">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Otomatik algıla</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Toplam puan</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="toplamPuan">
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Kitapçık sayısı</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="kitapcikSayisi">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>85</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="PlaceholderText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="128">
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>85</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="PlaceholderText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="128">
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>190</red>
                    <green>190</green>
                    <blue>190</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="PlaceholderText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="128">
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Öğrenci Sayısı</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ogrenciSayisi">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>85</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>85</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="WindowText">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>190</red>
                    <green>190</green>
                    <blue>190</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="dosyaAc">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Dosya Aç</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dersin Adı    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dersinAdi"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="notlarTablo">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="ogrnoSirala">
          <property name="text">
           <string>Öğrenci No'ya göre sırala</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="notSirala">
          <property name="text">
           <string>Nota göre sırala(azalan)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pusulayaUygun">
          <property name="text">
           <string>Pusulaya uygun(Öğr.No., Not)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="exceleAktar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string>Excel'e aktar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>dersinAdi</tabstop>
  <tabstop>notlarTablo</tabstop>
  <tabstop>dosyaAc</tabstop>
  <tabstop>otomatikAlgila</tabstop>
  <tabstop>toplamPuan</tabstop>
  <tabstop>exceleAktar</tabstop>
  <tabstop>ogrnoSirala</tabstop>
  <tabstop>notSirala</tabstop>
  <tabstop>pusulayaUygun</tabstop>
  <tabstop>soruSayisi</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
